addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -542,6 +542,7 @@ var SnippetEditor = Widget.extend({
         this.styles = {};
         this.selectorSiblings = [];
         this.selectorChildren = [];
+        this.excludeDropInSelector = [];
```

Comment:
Array variable names should use the plural form (so good that your variable name is clearer that the stupid `selectorSiblings` and `selectorChildren` which should really be `siblingSelectors` and `childSelectors` or even better `dropNearSelectors` and `dropInSelectors` but use `excludeDropInSelector**s**`) :+1: